src/wmnds/patterns/footer/_footer.scss

```diff
@@ -21,4 +21,169 @@
       display: block;
     }
   }
+
+  // footer v2 only styles
+  &--v2 {
+    padding: $size-lg $size-xs;
+
+    .wmnds-btn {
+      display: block;
+      color: get-color(primary);
+      background-color: $white;
+    }
+
+    .wmnds-footer__link {
+      color: currentColor;
+      font-weight: normal;
+      text-decoration: none;
+
+      &:hover {
+        color: currentColor;
+        text-decoration: underline;
+      }
+    }
+  }
+
+  @media screen and (max-width: $breakpoint-md - 1) {
+    .wmnds-footer__updates-col {
+      padding-bottom: $size-md * 1.25;
+
+      &:not(:last-child) {
+        margin-bottom: $size-xsm;
+        border-bottom: 1px solid get-color(secondary, 70);
+      }
+
+      p {
+        font-size: 0.937rem;
+      }
+    }
+  }
+
+  .wmnds-footer__heading {
+    margin-bottom: $size-md * 1.25;
+    padding-bottom: $size-md * 1.125;
+    border-bottom: 1px solid get-color(secondary, 70);
+
+    @media screen and (max-width: $breakpoint-md - 1) {
+      margin-bottom: $size-md;
+      font-weight: normal;
+
+      &.wmnds-collapse-heading {
+        margin: 0;
+        padding: $size-xsm 0;
+        border: 0;
+        font-family: $x-font-family;
+        font-size: $size-md * 1.125;
+      }
+    }
+
+    @media screen and (min-width: $breakpoint-md) {
+      padding-bottom: $size-md * 1.25;
+    }
+  }
+
+  //
+  .wmnds-footer__menu {
+    margin: 0;
+    padding: 0;
+
+    li {
+      margin-bottom: $size-sm;
+      list-style: none;
+    }
+  }
+
+  @media screen and (min-width: $breakpoint-md) {
+    .wmnds-grid {
+      justify-content: space-between;
+    }
+
+    .wmnds-col-lg-1-3 {
+      max-width: 30%;
+      margin-bottom: 2rem;
+    }
+  }
+
+  // styles for the bottom section of the footer
+  .wmnds-footer__bottom {
+    @media screen and (max-width: $breakpoint-md - 1) {
+      flex-direction: column-reverse;
+    }
+    margin-top: $size-xsm;
+    padding-top: $size-md;
+    border-top: 1px solid get-color(secondary, 70);
+
+    .wmnds-footer__link,
+    .wmnds-footer__copyright {
+      margin: 0;
+      font-family: $x-heading-font-family;
+    }
+  }
+
+  // menu styling for the bottom section
+  .wmnds-footer__bottom-menu {
+    margin: 0 0 $size-lg;
+    padding: 0;
+
+    li {
+      margin-bottom: $size-sm;
+      list-style: none;
+
+      a {
+        display: block;
+        white-space: nowrap;
+      }
+    }
+
+    @media screen and (min-width: $breakpoint-md) {
+      display: flex;
+      justify-content: flex-end;
+
+      li {
+        margin-bottom: 0;
+
+        &:not(:last-child) {
+          padding-right: $size-lg;
+        }
+      }
+    }
+  }
+}
+
```

Comment:
Can you add a class so that when you are selecting text in the footer with a mouse it is visible to the user.
By default it is:
```css
// Text highlighting color
::-moz-selection {
  // Code for Firefox
  color: $white;
  background: get-color(primary);
}

::selection {
  color: $white;
  background: get-color(primary);
}`